frontend/packages/dev-console/src/components/pipelines/modals/common/PipelineParameterSection.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { FieldArray } from 'formik';
+import { TextInputTypes } from '@patternfly/react-core';
+import { InputField } from '@console/shared';
+import { PipelineParam } from '../../../../utils/pipeline-augment';
+import FormSection from '../../../import/section/FormSection';
+
+type ParametersSectionProps = {
+  parameters: PipelineParam[];
+};
+
+const PipelineParameterSection: React.FC<ParametersSectionProps> = ({ parameters }) => (
+  <FieldArray
```

Comment:
Since no array helpers are being used here and based on this comment https://github.com/openshift/console/pull/5032#discussion_r408342466, you can remove `FieldArray` from here.

Reply:
I wanna say I didn't touch this... this was created around my first few weeks here at Red Hat  (and not really modified since I think). It was done by someone else on the team. I started a refactor several weeks back in order to reuse items between the two modals and it was probably a bad idea.

In the process of trying to cut everything out that is not going to be used. Sorry for the confusion.

Reply:
Yeah seeing all the other commits I thought you were doing the refactoring of the older code and hence suggested the change. Probably we should have created a tech debt for this and handled it later. The PR has become quite huge. :stuck_out_tongue:

Reply:
Yeah, I stacked them to reuse small parts... it was a short sighted decision on the weekend to close off on everything. Hoping to only copy the parts that need to overlap and significantly reduce the size of this PR and my other one.